helm/endpoint-copier-operator/templates/rbac/role.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: manager-role
+  name: endpoint-copier-operator
```

Comment:
Shouldn't this also be templated with `{{ include "endpoint-copier-operator.fullname" . }}` as that is how it is referred in the `ClusterRoleBinding`?